dist/mixins/carousel/ds6/carousel-mixins.less

```diff
@@ -20,12 +21,17 @@
     .icon--chevron-right,
     .icon--chevron-right-small {
         margin-left: 2px;
-        margin-top: -3px;
     }
 
     .icon--chevron-left,
     .icon--chevron-left-small {
         margin-left: -2px;
-        margin-top: -3px;
+    }
+
+    .icon--chevron-right,
```

Comment:
Again though, can't it just be `.icon` instead of this group?

Reply:
It's this way because the margins are different per side. One moves left 2px, and one moves right 2px.

Unless I'm not understanding what you're suggesting.

Reply:
Oh ... this is ... not correct. Let me see why this is here. I had removed it. (Sorry, I was confused for a moment.)

Reply:
Removed. It was an artifact of a bad build.